### DISCUSSION TITLE
Can you provide guidance on optimal resource allocation?

### DISCUSSION THREAD

**Discussion by Midnighter** [jotech/gapseq#74] [Q&A]
Hi,

I was wondering if you can provide some information on computational resources (CPU cores and memory) that you expect or that can be assigned to the individual steps of the [reconstruction workflow](https://gapseq.readthedocs.io/en/latest/usage/basics.html) in order to improve performance? And can I specify the number of threads/processes that gapseq will create (I only see the `-k` option to turn off parallel computation)?

I guess some of the questions in #59 also went in this direction.

**Comment by jotech** [comment]
hi @Midnighter, good point this information is missing you are right!

Needed resources depend a bit on the incoming genome file but in general I would recommend to run gapseq on computer cluster with one core and 3-5gb memory per input genome. Right now, this works best and scales reasable with several of hundreds of reconstructions in batch mode.
We try to come up with some more elaborated recommendations!
